portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -120,8 +120,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             response.sendRedirect("/contact.html");
             return;
         }
+
+        // Get the name input from the form.
+        String name = getParameter(request, "user-name", null);
+        // If the name field was left blank, change it to Anonymous
+        if (name.equals("")) {
+            name = "Anonymous";
```

Comment:
Couldn't you set the the default value as "Anonymous" when calling getParameter instead?

Reply:
Yeah I tried doing that. But the thing was that getParameter check if the parameter is null but, in the case of this form, when no name has been inserted the parameter will be "" and not null. That was why my getParameter was not doing the job.